tab error

If I type /sfcalc(SPACE) it gives an error. the error continues with every other letter regarding tab complete 
![image](https://github.com/Seggan/SFCalc/assets/29830635/77f673e3-2897-4f97-9885-c6a80b352a1a)
```[
13:51:50 ERROR]: Exception when 100petr attempted to tab complete sfcalc st
org.bukkit.command.CommandException: Unhandled exception during tab completion for command '/sfcalc st' in plugin SFCalc vDev - 41
at org.bukkit.command.PluginCommand.tabComplete(PluginCommand.java:150) ~[purpur-api-1.20.2-R0.1-SNAPSHOT.jar:?]
at org.bukkit.command.Command.tabComplete(Command.java:93) ~[purpur-api-1.20.2-R0.1-SNAPSHOT.jar:?]
at org.bukkit.command.SimpleCommandMap.tabComplete(SimpleCommandMap.java:253) ~[purpur-api-1.20.2-R0.1-SNAPSHOT.jar:?]
at org.bukkit.craftbukkit.v1_20_R2.CraftServer.tabCompleteCommand(CraftServer.java:2556) ~[purpur-1.20.2.jar:git-Purpur-2095]
at org.bukkit.craftbukkit.v1_20_R2.CraftServer.tabComplete(CraftServer.java:2528) ~[purpur-1.20.2.jar:git-Purpur-2095]
at org.bukkit.craftbukkit.v1_20_R2.command.BukkitCommandWrapper.getSuggestions(BukkitCommandWrapper.java:74) ~[purpur-1.20.2.jar:git-Purpur-2095]
at com.mojang.brigadier.tree.ArgumentCommandNode.listSuggestions(ArgumentCommandNode.java:71) ~[brigadier-1.1.8.jar:git-Purpur-2095]
at com.mojang.brigadier.CommandDispatcher.getCompletionSuggestions(CommandDispatcher.java:602) ~[purpur-1.20.2.jar:?]
at com.mojang.brigadier.CommandDispatcher.getCompletionSuggestions(CommandDispatcher.java:582) ~[purpur-1.20.2.jar:?]
at net.minecraft.server.network.ServerGamePacketListenerImpl.lambda$handleCustomCommandSuggestions$5(ServerGamePacketListenerImpl.java:831) ~[?:?]
at net.minecraft.server.TickTask.run(TickTask.java:18) ~[purpur-1.20.2.jar:git-Purpur-2095]
at net.minecraft.util.thread.BlockableEventLoop.doRunTask(BlockableEventLoop.java:153) ~[?:?]
at net.minecraft.util.thread.ReentrantBlockableEventLoop.doRunTask(ReentrantBlockableEventLoop.java:24) ~[?:?]
at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:1351) ~[purpur-1.20.2.jar:git-Purpur-2095]
at net.minecraft.server.MinecraftServer.d(MinecraftServer.java:193) ~[purpur-1.20.2.jar:git-Purpur-2095]
at net.minecraft.util.thread.BlockableEventLoop.pollTask(BlockableEventLoop.java:126) ~[?:?]
at net.minecraft.server.MinecraftServer.pollTaskInternal(MinecraftServer.java:1328) ~[purpur-1.20.2.jar:git-Purpur-2095]
at net.minecraft.server.MinecraftServer.pollTask(MinecraftServer.java:1321) ~[purpur-1.20.2.jar:git-Purpur-2095]
at net.minecraft.util.thread.BlockableEventLoop.managedBlock(BlockableEventLoop.java:136) ~[?:?]
at net.minecraft.server.MinecraftServer.waitUntilNextTick(MinecraftServer.java:1299) ~[purpur-1.20.2.jar:git-Purpur-2095]
at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1187) ~[purpur-1.20.2.jar:git-Purpur-2095]
at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:320) ~[purpur-1.20.2.jar:git-Purpur-2095]
at java.lang.Thread.run(Thread.java:833) ~[?:?]
Caused by: java.lang.NullPointerException: Cannot invoke "io.github.thebusybiscuit.slimefun4.api.items.SlimefunItem.isHidden()" because the return value of "io.github.thebusybiscuit.slimefun4.api.items.SlimefunItem.getById(String)" is null
at io.github.seggan.sfcalc.CalcCommand.lambda$onTabComplete$1(CalcCommand.java:83) ~[SFCalc.jar:?]
at io.github.seggan.sfcalc.errorreporter.ErrorReporter.executeOrElseReport(ErrorReporter.java:128) ~[SFCalc.jar:?]
at io.github.seggan.sfcalc.errorreporter.ErrorReporter.executeOrElseReport(ErrorReporter.java:140) ~[SFCalc.jar:?]
at io.github.seggan.sfcalc.CalcCommand.onTabComplete(CalcCommand.java:74) ~[SFCalc.jar:?]
at org.bukkit.command.PluginCommand.tabComplete(PluginCommand.java:138) ~[purpur-api-1.20.2-R0.1-SNAPSHOT.jar:?]
... 22 more
```
![image](https://github.com/Seggan/SFCalc/assets/29830635/df58a0e3-417a-4573-97fd-8febe1a18f7d)
![image](https://github.com/Seggan/SFCalc/assets/29830635/77c4c5d3-eeca-4b56-a25b-28b33dd42bf3)

--- a/src/main/java/io/github/seggan/sfcalc/CalcCommand.java
+++ b/src/main/java/io/github/seggan/sfcalc/CalcCommand.java
@@ -74,13 +74,15 @@ public class CalcCommand implements TabExecutor {
         SFCalc.REPORTER.executeOrElseReport(() -> {
             if (ids.isEmpty()) {
                 for (SlimefunItem item : Slimefun.getRegistry().getEnabledSlimefunItems()) {
-                    ids.add(item.getId().toLowerCase(Locale.ROOT));
+                    if (!item.isHidden()) {
+                        ids.add(item.getId().toLowerCase(Locale.ROOT));
+                    }
                 }
             }
 
             if (args.length == 1) {
                 for (String id : ids) {
-                    if (id.contains(args[0].toLowerCase(Locale.ROOT)) && !SlimefunItem.getById(id).isHidden()) {
+                    if (id.contains(args[0].toLowerCase(Locale.ROOT))) {
                         tabs.add(id);
                     }
                 }

